base16-apathy.reg

```diff
@@ -1,25 +1,72 @@
 Windows Registry Editor Version 5.00
 
-[HKEY_CURRENT_USER\Software\SimonTatham\PuTTY\Sessions\Base16-Apathy]
-"Colour0"="95,156,146"
+; Base16 Apathy
+; schema by Jannik Siebert (https://github.com/janniks)
+[HKEY_CURRENT_USER\Software\SimonTatham\PuTTY\Sessions\apathy]
```

Comment:
It's been a while since this commit. I was wondering why this line change?

Reply:
My 2 cents:

  * I like the new comment 👍 
  * I'd personally prefer to retain the old name, I.e. retain base16 prefix.